api-tester/build.gradle

```diff
@@ -14,4 +14,5 @@ dependencies {
     implementation project(path: ':public')
     implementation project(path: ':purchases')
     implementation "androidx.annotation:annotation:$annotationVersion"
+    implementation project(path: ':feature:google')
```

Comment:
hmmm our API testers are meant to only test public APIs, did we intentionally make GooglePurchaseOption public?

Reply:
ah okay i see below that we had `GoogleStoreProduct` exposed already...so maybe not part of this PR, but we might want to think through whether the customer needs to know about store-specific implementations

Reply:
Ahhh, so... @vegaro asked a similar thing below. `AmazonStoreProduct` is in the Amazon feature so I put the `GoogleStoreProduct` in the Google feature 🤷‍♂️ 

Are these not accessible to developers at all? Or are they accessible but only if the developers import the feature specifically?

Reply:
So, we currently release 2 dependencies. One is the main SDK which includes every module except `feature/amazon`, and another is an extra library for amazon corresponding to the `feature/amazon` module. If you want to use Amazon, you need to import both dependencies.

So the main SDK has a tree of modules but we currently only expose public methods in the `public` and `purchases` modules as part of the API. The other modules are internal dependencies that devs are not supposed to access. So this means that whatever we want devs to access, should have public visibility and be part of the `public` or `purchases` modules.